flake.nix

```diff
@@ -143,20 +150,21 @@
         '';
       };
     in
-    {
-      packages = {
-        default = releaseBuild;
+    rec {
+      packages = rec {
+        default = release;
+        release = releaseBuild;
         debug = debugBuild;
       };
       checks.default = testBuild;
       apps = {
         assigner = {
           type = "app";
-          program = "${self.packages.${system}.default}/bin/assigner";
+          program = "${packages.default}/bin/assigner";
         };
         block_gen = {
           type = "app";
-          program = "${self.packages.${system}.default}/bin/block_gen";
+          program = "${packages.default}/bin/block_gen";
         };
       };
       devShells.default = makeDevShell;
```

Comment:
Please add dev shell with debug input packages. I think it's even more essential then having debug package here, since you commonly need to debug tests, not binaries.

Reply:
Sure, you're right, I'll add it. Actually that's my mistake: I thought I've made dev shell debug by default.

Reply:
Done